namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/resources/rds.tf

```diff
@@ -0,0 +1,36 @@
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+module "dps_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.4"
```

Comment:
The RDS module is now at version 4.5

Please increase the ref value, and check the example to ensure you have configured all necessary parameters.